tools/accuracy_checker/accuracy_checker/annotation_converters/amazon.py

```diff
@@ -0,0 +1,327 @@
+"""
+Copyright (c) 2018-2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import pickle, random
+from pathlib import Path
+import numpy as np
+
+from ..representation import ClassificationAnnotation
+from ..config import NumberField, StringField, PathField, BoolField
+from ..utils import get_path
+from .format_converter import BaseFormatConverter
+from .format_converter import ConverterReturn
+
+def unicode_to_utf8(d):
+    return dict((key, value) for (key,value) in d.items())
```

Comment:
hmm... looks like function name is not correspond its behaviour.
Also not sure that we need function wrapper for single line operation